uni/lib/model/entities/library_occupation.dart

```diff
@@ -0,0 +1,75 @@
+/// Overall occupation of the library
+class LibraryOccupation {
+  int occupation = 0;
+  int capacity = 0;
+  List<FloorOccupation> floors = [];
+
+  LibraryOccupation(this.occupation, this.capacity);
+
+  void addFloor(FloorOccupation floor) {
+    floors.add(floor);
+    occupation += floor.getOccupation();
+    capacity += floor.getCapacity();
+  }
+
+  int getPercentage() {
+    if (capacity == 0) return 0;
+    return (occupation * 100 / capacity).round();
+  }
+
+  int getOccupation() {
+    return occupation;
+  }
+
+  int getCapacity() {
+    return capacity;
+  }
+
+  List<FloorOccupation> getFloors() {
+    return floors;
+  }
```

Comment:
In order to correctly define a getter in dart:
return_type get field_name{
    ...
}
https://www.geeksforgeeks.org/getter-and-setter-methods-in-dart/
However we don't need to implement logic in the getters, it's only acessing the variable. So we can get rid of the getters.